ZIGSIMPlus/Views/CommandSelectionViewController.swift

```diff
@@ -123,18 +120,35 @@ extension CommandSelectionViewController: UITableViewDataSource {
            cell.detailButton.isHidden = true
         }
 
-        // Make an adjustment table view screen
         if AppSettingModel.shared.isActiveByCommand[Command.allCases[indexPath.row]] == true {
             cell.commandOnOff.isOn = true
         } else {
             cell.commandOnOff.isOn = false
         }
         
-        // Set cell's UI design
+        let mediator = CommandAndServiceMediator()
+        if mediator.isAvailable(Command.allCases[indexPath.row]){
+            setAvailability(cell:cell, isAvailability: true)
+        } else {
+            setAvailability(cell:cell, isAvailability: false)
+        }
+
         cell.initCell()
         
         return cell
     }
+    
+    func setAvailability(cell:StandardCell ,isAvailability: Bool) {
```

Comment:
`isAvailability` should be renamed to `isAvailable`(both parameter and method name).
And in this came, when isAvailable is contained in the method name, external parameter name should be empty to avoid meaning duplication.

Reply:
レビューいただきありがとうございます。
修正しました。
https://github.com/1-10/ZIGSIMPlus_iOS_Public/blob/61a6bfdab8a2deba959c16b93f5c08a900982494/ZIGSIMPlus/Views/CommandSelectionViewController.swift#L141

Reply:
先程の口頭でのご指摘を受け、再度修正しました。

https://github.com/1-10/ZIGSIMPlus_iOS_Public/blob/c0c2fa2220bee9e8e2e1f29fd2d10930106ce716/ZIGSIMPlus/Views/CommandSelectionViewController.swift#L141